pkg/flow/temporary.go

```diff
@@ -195,23 +192,8 @@ func (im *instanceMemory) Raise(ctx context.Context, err *derrors.CatchableError
 }
 
 func (im *instanceMemory) RetrieveSecret(ctx context.Context, secret string) (string, error) {
-	var resp *secretsgrpc.SecretsRetrieveResponse
-
-	ns := im.cached.Namespace.ID.String()
-
-	resp, err := im.engine.secrets.client.RetrieveSecret(ctx, &secretsgrpc.SecretsRetrieveRequest{
-		Namespace: &ns,
-		Name:      &secret,
-	})
-	if err != nil {
-		s := status.Convert(err)
-		if s.Code() == codes.NotFound {
-			return "", derrors.NewUncatchableError("direktiv.secrets.notFound", "secret '%s' not found", secret)
-		}
-		return "", derrors.NewInternalError(err)
-	}
-
-	return string(resp.GetData()), nil
+	// TODO: yassir, fix secrets here.
```

Comment:
This needs to be fixed. If you implement an end-to-end test that uses secrets within a workflow, they will fail without this.